examples/launch_huggingface_classification.py

```diff
@@ -77,7 +77,7 @@
         config_space, mode=mode, metric=metric, random_seed=random_seed
     )
 
-    stop_criterion = StoppingCriterion(max_wallclock_time=3600)
+    stop_criterion = StoppingCriterion(max_wallclock_time=3000) # wall clock time can be increased to 1 hour for more performance
```

Comment:
Why setting a new constant here? Setting 3000 seconds is fine with me but the comment is then not in sync.

Reply:
Same rationale as previous instance